_inc/client/components/module-settings/index.jsx

```diff
@@ -40,6 +43,40 @@ export let SharedaddySettings = React.createClass( {
 SharedaddySettings = ModuleSettingsForm( SharedaddySettings );
 
 export let RelatedPostsSettings = React.createClass( {
+	renderPreviews() {
+		const show_headline = this.props.getOptionValue( 'show_headline' );
+		const show_thumbnails = this.props.getOptionValue( 'show_thumbnails' );
+		const previews = [ {
+			url: 'https://jetpackme.files.wordpress.com/2014/08/1-wpios-ipad-3-1-viewsite.png?w=350&h=200&crop=1',
+			text: __( 'Big iPhone/iPad Update Now Available' )
+		}, {
+			url: 'https://jetpackme.files.wordpress.com/2014/08/wordpress-com-news-wordpress-for-android-ui-update2.jpg?w=350&h=200&crop=1',
+			text: __( 'The WordPress for Android App Gets a Big Facelift' )
+		}, {
+			url: 'https://jetpackme.files.wordpress.com/2014/08/videopresswedding.jpg?w=350&h=200&crop=1',
+			text: __( 'Upgrade Focus: VideoPress For Weddings' )
+		} ];
+
+		return (
+			<div className="related-posts-settings_preview_container">
+				{
+					show_headline ?
+						<h3>Related</h3> :
```

Comment:
@oskosk this string **Related** must be translatable (see [this line](https://github.com/Automattic/jetpack/blob/master/modules/related-posts/jetpack-related-posts.php#L374) as verification of this statement).